test/rebar_pkg_SUITE.erl

```diff
@@ -198,6 +201,8 @@ good_disconnect(Config) ->
     CachedFile = filename:join(Cache, <<Pkg/binary, "-", Vsn/binary, ".tar">>),
     ?assert(filelib:is_regular(CachedFile)),
     {ok, Content} = file:read_file(CachedFile),
+    {_Package, CachePath} = rebar_pkg_resource:get_pkg_name_and_cache_path(State, Pkg, Vsn),
```

Comment:
CachePath = ?config(cache_dir, Config), would not suffice here?
rebar_pkg_resource:get_pkg_name_and_cache_path/3 function maybe not even needed.